chall04/ohakola.py

```diff
@@ -0,0 +1,50 @@
+# **************************************************************************** #
+#                                                                              #
+#                                                         :::      ::::::::    #
+#    ohakola.py                                         :+:      :+:    :+:    #
+#                                                     +:+ +:+         +:+      #
+#    By: ohakola <ohakola@student.hive.fi>          +#+  +:+       +#+         #
+#                                                 +#+#+#+#+#+   +#+            #
+#    Created: 2020/04/06 17:30:11 by ohakola           #+#    #+#              #
+#    Updated: 2020/04/06 17:58:25 by ohakola          ###   ########.fr        #
+#                                                                              #
+# **************************************************************************** #
```

Comment:
Please, do not add the 42 header for those challenges and try to keep it for the curriculum.